### DISCUSSION TITLE
Running GO as an older release?

### DISCUSSION THREAD

**Discussion by Echodonut** [geneontology/helpdesk#456] [Q&A] [answered]
Hello!

I would like to run GO enrichment analysis, but with the database as it was a few months ago (on April 22nd 2023 to be precise). Is there a way to do this?
I am asking because I did some analyses at that time, but I would like to add analyses for a few more sets of genes to compare and I cannot just redo them all with the current version.
I would be fine with having to learn how to run GO enrichment analysis locally. I imagine I would then plug in Data Archive that was updated in April this year (https://zenodo.org/record/7796232). But I don't know how to do these things, nor if there might be another way.

**Comment by Echodonut** [comment]
For clarification: it is not a viable option for me to redo my analyses with the current version, because I need to match it with already-completed analyses as part of my current project and another already published paper. I assumed it would be fairly easy to run a few extra sets on the older version of the database, considering the database is backed up frequently.

**Comment by kltm** [comment, marked as answer]
@Echodonut For older versions of the GO data from GO Central, you may find it easier to browse and use http://release.geneontology.org/ over the Zenodo archives, although they are the same--whatever works for you.

For repeating PANTHER enrichments (tagging @huaiyumi and @mugitty), you might want to contact them at https://github.com/pantherdb/Helpdesk/issues or [pantherfeedback@yahoo.com](mailto:pantherfeedback@yahoo.com).

**Reply by Echodonut** [reply]
Thanks @kltm ! I have reached out with a question on the PANTHER forum.
